clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -146,3 +149,42 @@ func (pc *PrometheusController) trySnapshotPrometheusDisk(pdName, snapshotName,
 	}
 	return err
 }
+
+func (pc *PrometheusController) deletePrometheusDiskIfEnabled() error {
+	if enabled, err := pc.isEnabled(); !enabled {
+		return err
+	}
+	if pc.diskMetadata.name == "" || pc.diskMetadata.zone == "" {
+		klog.Errorf("Missing zone or PD name, aborting deletion")
+		klog.Infof("PD name=%s, zone=%s", pc.diskMetadata.name, pc.diskMetadata.zone)
+		return fmt.Errorf("missing zone or PD name, aborting deletion")
+	}
+	// Delete Prometheus disk
+	return wait.Poll(
+		20*time.Second,
```

Comment:
Please extract this interval as const too.

Reply:
Done